### PR TITLE
Discord: Copy desktop file

### PR DESCRIPTION
**Pull request by TheDoctorsLife**
This way you can call discord in desktop menu's and what not.

### CODE REVIEW

**Comment by TheDoctorsLife** [comment]
I've updated this to move the package location as the desktop file expects Discord to be in the /usr/share folder rather than the /usr/share/applications folder.

**Comment by Vaelatern** [comment]
Patch the desktop file.
Binaries should not go in `/usr/share`

**Comment by TheDoctorsLife** [comment]
Alright done that, also stopped moving things that aren't needed.

File: srcpkgs/discord/template [hunk 1]
```diff
@@ -16,14 +16,25 @@ license="proprietary"
...
+	vcopy Discord ${_package_location}
+	vcopy libffmpeg.so ${_package_location}
+	vcopy libnode.so ${_package_location}
+	vcopy natives_blob.bin ${_package_location}
+	vcopy snapshot_blob.bin ${_package_location}
+	vcopy content_shell.pak ${_package_location}
+	vcopy icudtl.dat ${_package_location}
 	vmkdir /usr/bin
-	ln -sfr $DESTDIR/usr/lib/$pkgname/Discord $DESTDIR/usr/bin/discord
+	ln -sfr $DESTDIR${_package_location}/Discord $DESTDIR/usr/bin/Discord
```
Comment (Vaelatern) [inline srcpkgs/discord/template]: Any reason for the change to a capital letter? Just curious.

File: srcpkgs/discord/template [hunk 2]
```diff
@@ -16,14 +16,25 @@ license="proprietary"
...
+	vmkdir ${_package_location}
+	vcopy locales ${_package_location}
+	vcopy resources ${_package_location}
+	vcopy Discord ${_package_location}
+	vcopy libffmpeg.so ${_package_location}
+	vcopy libnode.so ${_package_location}
+	vcopy natives_blob.bin ${_package_location}
+	vcopy snapshot_blob.bin ${_package_location}
+	vcopy content_shell.pak ${_package_location}
+	vcopy icudtl.dat ${_package_location}
```
Comment (Vaelatern) [inline srcpkgs/discord/template]: Make this easier to understand
```
for item in locales resources DIscord libffmpeg.so libnode.so natives_blon.bin snapshot_blob.bin content_shell.pak icudtl.dat; do
      vcopy "${item}" "${_package_location}"
done

File: srcpkgs/discord/template (hunk 1 above)
Comment (TheDoctorsLife) [inline srcpkgs/discord/template]: The binary is a capital letter, so just matching Void's style guide.

**Comment by TheDoctorsLife** [comment]
So I forgot to push my local changes upstream, let me fix it.

**Comment by TheDoctorsLife** [comment]
There we go, sorry about that.

**Pull request merged**